js/index.js

```diff
@@ -0,0 +1,17 @@
+function toggleModal1(modalID) {

```

Comment:
We have already wrote `toggleModal` function in _header.css_, do we need this duplicated code?

Reply:
removed from index.js toggleModal1